server/datastore/mysql/software.go

```diff
@@ -642,23 +642,17 @@ func (si *softwareIterator) Next() bool {
 }
 
 // AllSoftwareWithoutCPEIterator Returns an iterator for the 'software' table, filtering out
-// software entries with CPEs and from the platforms included in the 'excludedPlatforms' param.
-func (ds *Datastore) AllSoftwareWithoutCPEIterator(ctx context.Context, excludedPlatforms []string) (fleet.SoftwareIterator, error) {
+// software entries with CPEs and from the sources included in the 'excludedSources' param.
+func (ds *Datastore) AllSoftwareWithoutCPEIterator(ctx context.Context, excludedSources []string) (fleet.SoftwareIterator, error) {
```

Comment:
This changes vulnerability scanning using NVD such that all dep and rpm packages are excluded, presumably because the OVAL scanner will do that. 


However, I think this means that some deb packages will not be scanned at all. For example, a deb package installed on debian.

Reply:
True, but we don't officially support Debian yet (see https://github.com/fleetdm/fleet/issues/7075)